### PR TITLE
Task-82-nav02

### PR DESCRIPTION
**Pull request by emmebravo**
## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Description

Issue #82

### CODE REVIEW

File: packages/client/src/components/styled/NavStyle.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    width: 50%;
+    height: 10%;
+    top: 45%;
+  }
+
+  &::before {
+    transform: ${(props) => (props.clicked ? "rotate(0)" : "rotate(90deg)")};
+  }
+  &::after {
+    transform: ${(props) => (props.clicked ? "rotate(0)" : "rotate(0)")};
```
Comment (indifferentghost) [inline packages/client/src/components/styled/NavStyle.js:92]: Does this need a props clicked if it's going to be `rotate(0)` either way?

File: packages/client/src/components/CreatePost.js [hunk 2]
```diff
@@ -46,6 +47,7 @@ function CreatePost() {
 
   return (
     <>
+      <Navbar isAdmin={isAdmin}></Navbar>
```
Comment (indifferentghost) [inline packages/client/src/components/CreatePost.js:50]: At some point of time we should probably lift these up to the app component, no reason for rendering these differently so many times.

I'll make a ticket for that.

**Review by indifferentghost — CHANGES_REQUESTED** [review]

File: packages/client/src/components/styled/NavStyle.js (hunk 1 above)
Comment (emmebravo) [reply packages/client/src/components/styled/NavStyle.js:92]: if I change the `rotate(0)` the + becomes an equal sign and out of frame. I'll look into it.

File: packages/client/src/Feed.js [hunk 3]
```diff
@@ -31,23 +32,26 @@ const Feed = () => {
...
-            key={post._id}
-            user={user}
-            setEditSubmitted={setEditSubmitted}
-          />
-        );
-      })}
-      <Footer isAdmin={isAdmin} />
-    </FeedDiv>
+    <>
+      <Navbar isAdmin={isAdmin}></Navbar>
```
[the remark this answers is not in the document]
Comment (emmebravo) [reply packages/client/src/Feed.js:36]: I had placed `Navbar` in `./App`, but it shows up in certain places we don't want, like the `login` page. Should I pass `Navbar` as `children`?

**Comment by emmebravo** [comment]
the whole button is now an svg, but I couldn't get it to `transform` when clicked from `+` to `x`; considered it not a priority, but let me know if it's otherwise

File: packages/client/src/Feed.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (indifferentghost) [reply packages/client/src/Feed.js:36]: Probably should then have two different layout wrappers,
```jsx
const excludeNav = [
  '/login'
];
function Layout({ children }) {
  const { pathname } = useLocation();
  if (excludeNav.includes(pathname)) {
    return <>{children}<Footer /></>;
  }
  return <><Navbar/>{children}<Footer/></>
}
```

**Review by indifferentghost — APPROVED** [review]

**Pull request merged**